ci/jenkins/jobs/macros.yaml

```diff
@@ -223,3 +223,38 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-vm.sh  --registry ${DOCKER_REGISTRY} --kubeconfig /var/lib/jenkins/.kube/config
+
+- builder:
+    name: builder-create-kind
```

Comment:
why add two job to add and delete kind cluster? I think we should create and delete the cluster during running e2e, conformance and networkpolicy job like what we did in capv cluster. Otherwise it's not easy to manage your kind environment.

Reply:
you are right we don't need two jobs to add and delete kind cluster, I have updated my code, thanks!

Reply:
> Why add two jobs to add and delete kind cluster? I think we should create and delete the cluster during running e2e, conformance and networkpolicy jobs just like what we did in capv cluster. Otherwise it's not easy to manage your kind environment.

you are right we don't need two jobs to add and delete kind cluster, I have updated my code, thanks!